### PR TITLE
EWPP-169: Set field instances as not translatable.

### PR DESCRIPTION
**Pull request by 22Alexandra**

### CODE REVIEW

File: tests/features/persistent-urls.feature [hunk 1]
```diff
@@ -41,8 +41,3 @@ Feature: Persistent URLs
     When I update alias of "News 1" node to "/alias1"
     And I go to "/news-2"
     Then I should see a persistent link for the node "News 1" pointing to "/en/alias1"
-    # Check link to first node with alias for another language.
-    When I update alias of "News 1" node to "/alias1_fr" for "French"
-    And I update alias of "News 2" node to "/news2_fr" for "French"
-    And I go to "/fr/news2_fr"
-    Then I should see a persistent link for the node "News 1" pointing to "/fr/alias1_fr"
```
Comment (nagyad) [inline tests/features/persistent-urls.feature:47]: Can we move this test to EWCMS instead of removing it?

Comment (22Alexandra) [reply tests/features/persistent-urls.feature:47]: We generate automatically the aliases in ewcms.

**Review by nagyad — DISMISSED** [review]

**Review by imanoleguskiza — DISMISSED** [review]

**Review by imanoleguskiza — DISMISSED** [review]

**Review by nagyad — DISMISSED** [review]

**Review by nagyad — DISMISSED** [review]

**Comment by 22Alexandra** [comment]
The title field of Project ct is still translatable (core.base_field_override.node.oe_project.title.yml).

**Review by 22Alexandra — COMMENTED** [review]
The title field of Project ct is still translatable (core.base_field_override.node.oe_project.title.yml).

**Review by nagyad — DISMISSED** [review]

**Review by imanoleguskiza — DISMISSED** [review]

**Review by drishu — CHANGES_REQUESTED** [review]
consultation.feature & person.feature also need to be updated to the title change.

File: tests/features/publication.feature [hunk 2]
```diff
@@ -133,7 +133,7 @@ Feature: Publication content creation
     And the following General Contact entity:
       | Name          | A general contact |
     And the following "Publication" Content entity:
-      | Title                  | Publication demo page                 |
+      | Title             | Publication demo page                 |
```
Comment (drishu) [inline tests/features/publication.feature]: indentation could be better here but blocking for this is silly

**Review by drishu — DISMISSED** [review]

**Review by drishu — DISMISSED** [review]

File: modules/oe_content_publication/config/install/core.base_field_override.node.oe_publication.title.yml [hunk 3]
```diff
@@ -7,7 +7,7 @@ id: node.oe_publication.title
 field_name: title
 entity_type: node
 bundle: oe_publication
-label: Title
+label: Page title
```
Comment (ademarco) [inline modules/oe_content_publication/config/install/core.base_field_override.node.oe_publication.title.yml]: This is out of scope, if this is needed, please open a new issue and we will prioritize it with the PO.

**Review by ademarco — DISMISSED** [review]

**Review by nagyad — DISMISSED** [review]

File: tests/features/persistent-urls.feature [hunk 4]
```diff
@@ -39,8 +39,3 @@ Feature: Persistent URLs
     When I update alias of "News 1" node to "/alias1"
     And I go to "/news-2"
     Then I should see a persistent link for the node "News 1" pointing to "/en/alias1"
-    # Check link to first node with alias for another language.
-    When I update alias of "News 1" node to "/alias1_fr" for "French"
-    And I update alias of "News 2" node to "/news2_fr" for "French"
```
Comment (yenyasinn) [inline tests/features/persistent-urls.feature:45]: News is untranslatable because "At least one field needs to be translatable to enable News for translation.". So this step fails.

**Review by upchuk — DISMISSED** [review]

**Review by yenyasinn — APPROVED** [review]

**Review by upchuk — APPROVED** [review]

**Review by nagyad — APPROVED** [review]
